site-builder/tests/publish.rs

```diff
@@ -38,3 +38,34 @@ async fn publish_snake() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn json_publish_snake() -> anyhow::Result<()> {
+    let cluster = TestSetup::start_local_test_cluster().await?;
+    let directory = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .parent()
+        .unwrap()
+        .join("examples")
+        .join("snake");
+
+    let args = ArgsBuilder::default()
+        .with_command(Commands::Json {
+            command_string: Some(format!(
+                r#"{{
+            "config":"{}",
+            "command":{{
+                "publish":{{
+                    "directory":"{}",
+                    "epochs":1
+                }}
+            }}
+        }}"#,
```

Comment:
```suggestion
                r#"{{
            "config":"{}",
            "context": "{}" // added context 
            // ... add more here
            "command":{{
                "publish":{{
                    "directory":"{}",
                    "epochs":1
                }}
            }}
        }}"#,
```
suggestion: let's define explicitly as many options as possible so it can act as a reference for devs.

Reply:
I can add it to the Json docstring though.